src/object/obj_rpc.c

```diff
@@ -549,7 +567,10 @@ obj_reply_dtx_conflict_set(crt_rpc_t *rpc, struct dtx_conflict_entry *dce)
 	}
 	case DAOS_OBJ_RPC_PUNCH:
 	case DAOS_OBJ_RPC_PUNCH_DKEYS:
-	case DAOS_OBJ_RPC_PUNCH_AKEYS: {
+	case DAOS_OBJ_RPC_PUNCH_AKEYS:
```

Comment:
(style)  Possible switch case/default not preceeded by break or fallthrough comment